lib/beaker/dsl/install_utils/windows_utils.rb

```diff
@@ -5,14 +5,15 @@ module InstallUtils
       # This module contains methods useful for Windows installs
       #
       module WindowsUtils
-
         # Given a host, returns it's system TEMP path
+        #
         # @param [Host] host An object implementing {Beaker::Hosts}'s interface.
-        def get_temp_path(host)
-          # under CYGWIN %TEMP% may not be set
-          tmp_path = on(host, Command.new('ECHO %SYSTEMROOT%', [], { :cmdexe => true }))
-          tmp_path.output.gsub(/\n/, '') + '\\TEMP'
+        #
+        # @return [String] system temp path
+        def get_system_temp_path(host)
```

Comment:
I actually like having the alias as a pointer to something that should be deprecated in the future.

Also, we already have a pattern in beaker where `host.tmpdir` should give you a user's temp dir.  Making this change just allows there to be two ways to get a temp directory for a user (even if `get_temp_path`'s would just be pointing to `host.tmpdir`), which is something I'd like to avoid.